docs/management_api.yml

```diff
@@ -255,6 +326,45 @@ paths:
         500:
           $ref: '#/components/responses/InternalServerError'
 
+  /devices/{id}/upload:
+    post:
```

Comment:
idle question, do we consider overwriting a file with PUT? what do we do in general when a file exists?

Reply:
the `/upload` endpoint overwrites the files. I wasn't sure about PUT vs. POST, but as we accept a multipart/form-data payload, I thought POST is more appropriate. @alfrunes what do you think?

Reply:
According to the [specification](https://tools.ietf.org/html/rfc7231#section-4.3.4) it sounds more correct to use `PUT` here actually.